.github/workflows/windows-build.yml

```diff
@@ -2,7 +2,10 @@ on:
   push:
     branches:
       - master
-  pull_request:
+      - /feature.*/
```

Comment:
Should this be `/^feature.*/`?  Should it more precisely match Travis which includes `^feature-` and `^features/`?

Reply:
this regex will grab both 'features/' and 'feature-'

Reply:
I guess I'm just suggesting we do exactly the same thing in both Travis and GHA.  No reason to be subtly different.

Reply:
no worries!